plugin.rb

```diff
@@ -5,62 +5,81 @@
 # version: 0.1
 # authors: Neil Lalonde
 # url: https://github.com/discourse/discourse-saved-searches
+# transpile_js: true
 
 enabled_site_setting :saved_searches_enabled
 
 register_asset 'stylesheets/saved-searches.scss'
 
 after_initialize do
-
-  plugin = self
-
-  require_dependency File.expand_path('../app/jobs/regular/saved_search_notification.rb', __FILE__)
-  require_dependency File.expand_path('../app/jobs/scheduled/schedule_saved_searches.rb', __FILE__)
-
-  User.register_custom_field_type('saved_searches', :json)
-
-  add_to_serializer(:user, :saved_searches, false) do
-    (object.custom_fields['saved_searches'] || {})['searches']
-  end
-
-  add_to_serializer(:user, :include_saved_searches?, false) do
-    plugin.enabled? && scope.can_edit?(object)
-  end
-
   module ::SavedSearches
     class Engine < ::Rails::Engine
       engine_name 'saved_searches'
       isolate_namespace SavedSearches
     end
   end
 
-  require_dependency 'application_controller'
-
-  class ::SavedSearches::SavedSearchesController < ::ApplicationController
-    requires_plugin 'discourse-saved-searches'
+  require File.expand_path('../app/controllers/saved_searches_controller.rb', __FILE__)
```

Comment:
I think `require_relative` is a shortcut for this:

```suggestion
  require_relative 'app/controllers/saved_searches_controller.rb'
```